### ISSUE TITLE
[a11y-defect-3]: Download benefit letters container should not be a live region

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96352]
### Point of contact

Trevor Pierce

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

The Download VA Letters and Documents view has a live region DIV around the accordions where users can download their benefit letters. This live region causes each accordion to read their entire contents when opened. It's a bit too much information and could become confusing if a user expanded all accordions. Screenshot attached.

---
![Image](https://github.com/user-attachments/assets/c7bc1f26-2616-4e51-bbb5-a75a8b6b914c)

### Reproduction steps

1. Navigate to https://staging.va.gov/records/download-va-letters/letters/letter-list with a test user who has benefit letters. I tested with 54.
2. Turn on the screen reader of your choosing
3. Navigate to an accordion and press `Enter` to expand it
4. Verify the entire contents are read back after hearing "Accordion name, expanded"

### Proposed solution or next steps

Remove the `aria-live` region from the containing DIV. See the screenshot above for the attribute to remove.

### References, articles, or WCAG support

1. Understanding [SC 1.3.1](https://w3.org/TR/WCAG22#info-and-relationships): Info and Relationships (Level A)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
Let's get a video demo of the screen reader functionality uploaded to this ticket so we can verify quickly.  :+1: @1Copenut  @janechodance  @dysbo

**Comment by dysbo** [comment]
Verification video:
https://github.com/user-attachments/assets/6b76c00a-834e-474b-b7f3-81bd7b5f992c
